addons/hr_timesheet/views/project_task_portal_templates.xml

```diff
@@ -23,7 +23,7 @@
             </div>
         </xpath>
         <xpath expr="//div[@name='portal_my_task_allocated_hours']" position="after">
-            <div t-if="task.allocated_hours > 0 and allow_timesheets"><strong>Progress:</strong> <span t-field="task.progress"/>%</div>
+            <div t-if="task.allocated_hours > 0 and allow_timesheets"><strong>Progress:</strong> <span t-esc="task.progress * 100"/>%</div>
```

Comment:
Cannot we use the `percentage` widget here? 🤔 

```suggestion
            <div t-if="task.allocated_hours > 0 and allow_timesheets"><strong>Progress:</strong> <span t-esc="task.progress" t-options="{'widget': 'percentage'}"/></div>
```

I don't know if the widget exists in portal. 🤷🏼‍♂️

Reply:
it doesn't exist in portal, do you prefer importing it or keep the *100 and % ?

Reply:
We can keep your solution then